### ISSUE TITLE
One-Click Deploy to Heroku on Free Plan?

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/adoro#9]
In order to aid adoption, should we have a one-click deploy build plan for heroku?

**Comment by iteles** [comment]
Definitely! A _nice-to-have_ but a really great one that will certainly make a difference in adoption along with some nice tutorials and documentation (with a detailed troubleshooting section built up over time). :heart:

**Comment by sofer** [comment]
Probably best to assume that none of the first 20 deployments will be done by anyone outside FAC. A really great blogging platform that *we* can use and that is not WordPress is a great target to aim for. A bit of setup pain for early adopters is to be expected.
